extension/manifest.json

```diff
@@ -88,7 +88,7 @@
       "js": ["shared.js","keysocket-slacker.js"]
     },
     {
-      "matches": ["*://music.yandex.ru/*"],
+      "matches": ["*://music.yandex.ru/*", "*://music.yandex.by/*", "*://music.yandex.ua/*", "*://music.yandex.kz/*", "*://music.yandex.tr/*"],
```

Comment:
No, this change leads to an error: `Invalid value for 'content_scripts[9].matches[0]': Invalid host wildcard.` As I understand asterisk must on the left side of a wildcard.